tests/config.py

```diff
@@ -1,18 +1,18 @@
+from datetime import datetime
+from os.path import abspath, dirname, join as join_path
 import json
-import os
 import sys
 import pystac
-from datetime import datetime
 
-sys.path.append(os.path.dirname(
-    os.path.abspath(__file__)) + '/../gdal_subsetter/'
-                )
-from transform import HarmonyAdapter
-
-import harmony
 from harmony.message import Message
 from harmony.logging import setup_stdout_log_formatting
-from harmony.util import (CanceledException, config, create_decrypter)
+from harmony.util import config, Config, create_decrypter
+from osgeo import gdal
+from osgeo.osr import SpatialReference
+
+sys.path.append(dirname(abspath(__file__)) + '/../gdal_subsetter/')
+from transform import HarmonyAdapter
+
```

Comment:
I had to make a few changes to the file paths to make the tests run both on a local machine and in a Docker container. I've also cleaned up the test files a little bit.

Generally, I think we probably need to add some genuine unit tests to this repository, beyond the ones that test the service as a whole. Like the linting clean-up, this is probably something to do gradually and in bit-size pieces.